src/Plugin/GraphQL/DataProducer/Routing/RouteLoad.php

```diff
@@ -90,14 +94,16 @@ public function __construct(
    * Resolver.
    *
    * @param string $path
+   * @param string|null $language
+   *   The language code of the redirect.
    * @param \Drupal\Core\Cache\RefinableCacheableDependencyInterface $metadata
    *
    * @return \Drupal\Core\Url|null
    */
-  public function resolve($path, RefinableCacheableDependencyInterface $metadata) {
+  public function resolve(string $path, ?string $language, RefinableCacheableDependencyInterface $metadata) {
```

Comment:
Instead of allowing null, shouldn't the lang param have 'und' as default

`$language = Language::LANGCODE_NOT_SPECIFIED`

Reply:
Good point, this was done in #1328